azerothcore/azerothcore-wotlk
# fix(DB/creature,SAI): Silvermane Stalkers: grey -> black, no fade

Co-authored-by: @Branel

## Changes Proposed:
- Change the spawned model for **Silvermane Stalkers (2926)** from **grey (11418)** to **black (9562)** which matches the template
- Remove **fade** ability from SmartAI
- Unhide nameplate

## Issues Addressed:
- Closes #5847
- Closes https://github.com/chromiecraft/chromiecraft/issues/645

## SOURCE:
See #5847


## Tests Performed:
- SQL statements executed
- Tested in game


## How to Test the Changes:
- Execute the SQL statements from the patch
- Restart your `worldserver`
- Go to **Hinterlands**
- Check out the **Silvermane Stalkers** near **Skulk Rock**. They should be black now, nameplates should be visible without targeting them and they should no longer be stealthed/faded.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
